terracumber_config/tf_files/variables/PR-TEST-variable.tf

```diff
@@ -0,0 +1,166 @@
+// Mandatory variables for terracumber
+variable "URL_PREFIX" {
+  type = string
+  default = "Define in product tfvars"
+}
+
+// Not really used as this is for --runall parameter, and we run cucumber step by step
+variable "CUCUMBER_COMMAND" {
+  type = string
+  default = "Define in product tfvars"
+}
+
+variable "CUCUMBER_GITREPO" {
+  type = string
+  default = "Define in product tfvars"
+}
+
+variable "CUCUMBER_BRANCH" {
+  type = string
+  default = "Define in product tfvars"
+}
+
+variable "CUCUMBER_RESULTS" {
+  type = string
+  default = "/root/spacewalk/testsuite"
+}
+
+variable "MAIL_SUBJECT" {
+  type = string
+  default = "Define in product tfvars"
+}
+
+variable "MAIL_TEMPLATE" {
+  type = string
+  default = "Define in product tfvars"
+}
+
+variable "MAIL_SUBJECT_ENV_FAIL" {
+  type = string
+  default = "Failed acceptance tests on Pull Request: Environment setup failed"
+}
+
+variable "MAIL_TEMPLATE_ENV_FAIL" {
+  type = string
+  default = "Define in product tfvars"
+}
+
+variable "ENVIRONMENT" {
+  type = string
+}
+
+variable "MAIL_FROM" {
+  type = string
+  default = "galaxy-ci@suse.de"
+}
+
+variable "MAIL_TO" {
+  type = string
+  default = "galaxy-ci@suse.de"
+}
```

Comment:
I think that better we don't set as default emails the galaxy-ci mailing list, otherwise we will introduce too much noise to this mailing list.
Instead use, galaxy-noise, unless the user set their own email.